packages/strapi-versioning-plugin/src/services/Versioning.ts

```diff
@@ -0,0 +1,50 @@
+import { Context } from "koa";
+
+type Entity = any;
+type Data = Array<Entity>;
+
+const isModelExists = (ctx: Context): boolean =>
+  (ctx.params !== undefined && ctx.params.model !== undefined) ||
+  Object.values(global.strapi.models).find(
+    (el) => el.collectionName == ctx.url.split("/")[1],
+  ) !== undefined;
+
+const findUid = (ctx: Context): string =>
+  ctx.params.model
+    ? global.strapi.db.getModel(ctx.params.model).uid
+    : Object.values(global.strapi.models).find(
+        (el) => el.collectionName == ctx.url.split("/")[1],
+      )!.uid;
+
+const saveDataInDB = async (
+  data: Data,
+  model: string,
+  entity: Entity,
+): Promise<void> => {
+  const knexQueryBuilder = global.strapi.connections.default("versions");
+  knexQueryBuilder
+    .insert({
+      date: new Date().toISOString(),
+      content_type: model,
+      content: JSON.stringify(data),
+      entity_id: entity.id,
+      entity: JSON.stringify(entity),
+    })
+    .returning("*")
+    .toString();
+  const resp = await knexQueryBuilder.then();
```

Comment:
what is it? 
1. need to handle erorrs (did You test, when db is not responding?) 
2. doesn't need to return enything, so also doesn't need to call `.then()`

Reply:
then removed, errors handled by frontend, backend in case of error returning 500